### ISSUE TITLE
ProjectE (and its addon's) Changes

### ISSUE THREAD

**Issue by Rongmario** [EnderExtreme/Toxic-Garbage-Island#200]
## Changes:
1. Covalence loss = 0.5
2. Katar's DeathAura = 500DMG
3. Harvest Band Grass Growth turned on
4. Watch of Flowing Time -> Pedestal Bonus: 10 ticks instead of 18
5. Watch of Flowing Time will be able to speed up TEs as opposed of Time in a Bottle from RandomThings
6. Custom Mapping in TGILibrary + ProjectE Addons

### Ability Cooldown Changes:
1. Gem Chest explosion = 300 ticks (15 seconds)
2. Projectile = 120 ticks (6 seconds)

#### In Pedestal: 
1. Archangel Smite = 140 ticks (7 seconds)
2. Body Stone Healing = 80 ticks (4 seconds)
3. Evertide = 200 ticks (120 seconds)
4. Harvest Goddess Band = 60 ticks (3 seconds)
5. Ignite Ring = 40 ticks (2 seconds)
6. Life Stone Healing = 80 ticks (4 seconds)
7. Talisman of Repair = 20 ticks (1 second)
8. Soul Stone Healing = 20 ticks (1 second)
9. SWRG = 100 ticks (5 seconds)
10. Volcanite = 200 ticks (120 seconds)
11. Zero = 100 ticks (5 seconds)

**Comment by Rongmario** [comment]
Disabled covalence loss.
